src/gobapi/config.py

```diff
@@ -15,3 +15,8 @@
     'host': os.getenv("DATABASE_HOST_OVERRIDE", "localhost"),
     'port': os.getenv("DATABASE_PORT_OVERRIDE", 5406),
 }
+
+# see gobapi.services.registry
+API_INFRA_SERVICES = os.getenv(
+    "API_INFRA_SERVICES", ""
+).upper().split(",")
```

Comment:
Question: Should any value be set when running API outside docker?

Reply:
71b575010b042d34f07b61faad313bf8b48570c2